docs/modules/plugins/pages/plugin-azure-functions.adoc

```diff
@@ -0,0 +1,67 @@
+= Azure functions Plugin
+
+The plugin provides functionality to interact with https://azure.microsoft.com/en-us/services/functions/.
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-azure-functions', version: '{current-version}')
+----
+
+== Configuration
+
+=== Authentication
+
+By default, plugin's authentication depends on correct configure of following environment variables.
+
+
+. `AZURE_CLIENT_ID` - for Azure client ID
+. `AZURE_CLIENT_SECRET` - for client secret
+. `AZURE_CLIENT_CERTIFICATE_PATH` - for client certificate
+. `AZURE_TENANT_ID` - for Azure tenant ID
+. `AZURE_SUBSCRIPTION_ID` - Azure subscription ID
```

Comment:
Do all of them are needed? Or maybe we can define groups of variables specifying different authentication approaches